docs/data/data-grid/server-side-data/ServerSideDataGridTTL.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import {
+  DataGridPro,
+  GridDataSource,
+  SimpleServerSideCache,
+} from '@mui/x-data-grid-pro';
+import { useMockServer } from '@mui/x-data-grid-generator';
+import LoadingSlate from './LoadingSlateNoSnap';
+
+const lowTTLCache = new SimpleServerSideCache({ ttl: 1000 * 60 * 1 }); // 1 minute
```

Comment:
For demo purposes, I would set `ttl` to 5-10s for easier stale cache testing

Reply:
Makes sense 👍